src/genlab_bestilling/models.py

```diff
@@ -614,6 +614,11 @@ class SampleMarkerAnalysis(models.Model):
     marker = models.ForeignKey(f"{an}.Marker", on_delete=models.PROTECT)
     transaction = models.UUIDField(blank=True, null=True)
 
+    # NEW FIELDS for status tracking
```

Comment:
```suggestion
    # Fields for status tracking
```
It won't be "new fields" forever.